components/entries/IncompleteButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from 'react-i18next';
+import ButtonWithIcon from '../ButtonWithIcon';
+
+const IncompleteButton = () => {
```

Comment:
[non-blocking] [curious] Potentially, `CompleteButton` and `IncompleteButton` could be one component, with a state/complete prop instead - I suggest this because if you wanted to do a transition between the two or link their behaviour together in some way, they'd be in the same component and it's much easier at that point

Reply:
Good idea :)